plugins/snowflake/dbt/include/snowflake/macros/adapters.sql

```diff
@@ -35,13 +39,25 @@
     {% if enable_automatic_clustering and cluster_by_string is not none and not temporary  -%}
       alter table {{relation}} resume recluster;
     {%- endif -%}
+    -- add in comments
+
+    {% if relation_comment is not none -%}
+      {{ alter_relation_comment(relation, relation_comment) }}
+    {%- endif -%}
+
+    {% if column_comment is not none -%}
+      {{ alter_column_comment(relation, column_comment) }}
+    {%- endif -%}
 
 {% endmacro %}
 
 {% macro snowflake__create_view_as(relation, sql) -%}
   {%- set secure = config.get('secure', default=false) -%}
   {%- set copy_grants = config.get('copy_grants', default=false) -%}
   {%- set sql_header = config.get('sql_header', none) -%}
+  {%- set raw_persist_docs = config.get('persist_docs', {}) -%}
+  {%- set relation_comment = get_relation_comment(raw_persist_docs, model) -%}
+  {%- set column_comment = get_relation_column_comments(raw_persist_docs, model) -%}
```

Comment:
While it appears that we're not going to be able to set column comments on views, I _do_ think we can set a view-level comment! 

Can you add a call to `snowflake__alter_relation_comment` at the bottom of this macro to render the SQL required to add a view comment?